sr_system_info/src/sr_system_info/system_info.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python
+#
+# Copyright (C) 2020 Shadow Robot Company Ltd - All Rights Reserved. Proprietary and Confidential.
+# Unauthorized copying of the content in this file, via any medium is strictly prohibited.
+
+import argparse
+import psutil
+import rospy
+import rospkg
+import subprocess
+import yaml
+
+
+class SystemInfo(object):
+
+    def __init__(self):
+        self.rospack = rospkg.RosPack()
+
+    def collect(self):
+        self.values = {}
+        self.values["ros_root_path"] = rospkg.get_ros_root()
+        self.values["ros_package_paths"] = rospkg.get_ros_package_path().split(':')
+        self.values["src_package_paths"] = [i for i in self.values["ros_package_paths"] if "src" in i]
+        self.survey_packages()
+        self.survey_source_repos()
+        self.survey_system()
+        # rospy.loginfo(self.yaml())
```

Comment:
please remove commented out code

Reply:
Fixed in https://github.com/shadow-robot/sr_utils/pull/6/commits/321bb546d91af88f074b8164506d2429c7b1d232